assets/scss/pages/contact-page.scss

```diff
@@ -0,0 +1,109 @@
+.c-contact-page__section {
+    display: flex;
+    flex-direction: column;
+    padding: 24px;
+    gap: 24px;
+
+    @include bp(medium) {
+        padding: 48px;
+        gap: 48px;
+    }
+
+    @include bp(large) {
+        padding: 48px 80px;
+    }
+}
+
+.c-contact-page__title {
+    margin: 0;
+    color: $color-black;
+    font-family: $font-semi-bold;
+    font-size: 32px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.25;
+}
+
+.c-contact-form {
```

Comment:
@mgbelegu it would be better to make a separate component here, unless it's strictly related to this page, than I would go with `p-contact__form`.